pkg/fileutils/fileutils_unix.go

```diff
@@ -14,7 +14,7 @@ import (
 // reading it via /proc filesystem.
 func GetTotalUsedFds() int {
 	if fds, err := ioutil.ReadDir(fmt.Sprintf("/proc/%d/fd", os.Getpid())); err != nil {
-		logrus.Errorf("Error opening /proc/%d/fd: %s", os.Getpid(), err)
```

Comment:
This should be
```diff
+ logrus.Errorf("Error opening /proc/%d/fd: %v", os.Getpid(), err)
```

Reply:
That would (re)introduce stutter since the path is already part of `err`.

Reply:
@vrothberg ah my bad if comment is confusing PR does a `%s` for err just wanted to point we must use `%v` or `%w`.